### PR TITLE
Extend the task execution policy document to enable AWS SSM integration

### PR DESCRIPTION
**Pull request by Overbryd**
Allow fetching secure values from AWS SSM.

Example:

```terraform
module "fargate" {
  source = "git::https://github.com/Overbryd/terraform-aws-fargate.git?ref=feat/task-execution-policy-ssm"

  name = "backend"

  aws_ssm_allowed_parameters_arn = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter/backend_*"
```

### CODE REVIEW

**Comment by ivanmb** [comment]
+1 On this! Great combination of features

Thanks a lot

File: main.tf [hunk 1]
```diff
@@ -119,9 +119,38 @@ resource "aws_iam_role" "tasks" {
...
+  statement {
+    actions = [
+      "ssm:GetParameter*"
+    ]
+    resources = ["${var.aws_ssm_allowed_parameters_arn}"]
+  }
+
+  statement {
+    actions = [
+      "ssm:DescribeParameters",
```
Comment (jlsan92) [inline main.tf]: Only `ssm:GetParameters` is required as described here -> https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html

File: policies/ecs-task-execution-role-policy.json [hunk 2]
```diff
@@ -1,17 +0,0 @@
-{
```
Comment (jlsan92) [inline policies/ecs-task-execution-role-policy.json]: As you may have seen, all policies that are used in this module, are stored in the `policies` directory. Let's keep that design 🙏.

To include a computed value into the policy file, we can reference it by using the `template` provider.

Eg. https://github.com/strvcom/terraform-aws-fargate/blob/master/main.tf#L91-L99

**Comment by jlsan92** [comment]
Thank you. #48 merged 🚀

**Pull request closed without merging**